code.js

```diff
@@ -0,0 +1,58 @@
+//Kruskal's Algorithm
+function kruskal(graph) {
+
+    //Start with an empty tree T
+    let tree = [];
+    for (let i = 0; i < graph.length; i++) {
+        tree[i] = [];
+    }
+    //Build a priority-queue-type-thing
```

Comment:
Don't understand how you implemented this. Kruskal's algorithm processes edges in order of weight, so that's what I would expect to see in the priority queue. But it looks like you're storing something different?